website/common/locales/en/groups.json

```diff
@@ -114,8 +114,8 @@
     "gemAmountRequired": "A number of gems is required",
     "notAuthorizedToSendMessageToThisUser": "Can't send message to this user.",
     "privateMessageGiftIntro": "Hello <%= receiverName %>, <%= senderName %> has sent you ",
```

Comment:
It seems like `"privateMessageGiftSubscriptionMessage"` is no longer used in the code base. It is probably okay to delete that.